functions.php

```diff
@@ -728,6 +728,16 @@ function get_the_localization_language_by_website($site=""){
     $language['Vietnam'] = "Vietnamese";
     return $language[$site_name];
 }
+function posts_for_both_and_current_languages($postID, $current_lang = "en", $taxonomy ="language"){
+    $site_language = strtolower(get_localization_language_by_language_code($current_lang));
+    $terms = get_the_terms($postID, $taxonomy);
+    if ( empty($terms) && !is_wp_error( $terms )) {
+        return true;
+    }else if (has_term( "$site_language", $taxonomy, $postID )){
```

Comment:
Nope. the quotes should be removed, but i forget.  I will remove it and push the updated code soon. 
Anyways, I didn't get the error with it.